app/helpers/application_helper.rb

```diff
@@ -77,4 +77,16 @@ def time_period_class(item)
     return 'currently-valid' if item.currently_valid?
     return 'expired' if item.expired?
   end
+
```

Comment:
You might want to see how to integrate with Simple Form.

Big buttons for normal forms. Small buttons only for table/summary views.